client/views/app.html

```diff
@@ -80,6 +80,9 @@ <h5>Please choose a region below so we can know how horrible you are</h5>
       </div>
     </div>
     <answers></answers>
+    <div>
+      <chat></chat>
+    </div>
```

Comment:
Does the `div` around the `<chat>` directive play any role? I feel it can be moved to the template file of the directive.